### DISCUSSION TITLE
I havent recevied student developer pack benifits

### DISCUSSION THREAD

**Discussion by vineet-hub7** [community/community#66634] [GitHub Education]
### Hi everyone,

I applied for student developer pack benifits 3 days ago and i havent still recevied any email.
When i uploaded proof of identity card and clicked on process my application it just loaded to 100% and no confirmation or succesfull email i recevied for completing application

Please anyone guide me through

**Comment by arafajs** [comment]
If accepted, you should wait 48 - 72 hours for the benefits to be activated.

**Reply by vineet-hub7** [reply]
i havent recevied the acceptance confirmation
